client/src/pages/Login/useStyles.ts

```diff
@@ -2,20 +2,41 @@ import { makeStyles } from '@mui/styles';
 
 const useStyles = makeStyles(() => ({
   root: {
-    minHeight: '100vh',
+    // minHeight: '100vh',
     '& .MuiInput-underline:before': {
       borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
     },
   },
   authWrapper: {
```

Comment:
Most of these styles can be applied to the `Box` component direclty

Eg: `<Box display="flex" alignItems="center"....>`